### DISCUSSION TITLE
What process to follow in order to add product to toolchain list ?

### DISCUSSION THREAD

**Discussion by williamtcdns** [zephyrproject-rtos/sdk-ng#841] [Q&A] [answered]
There are multiple builds of 'Zephyr tools for Xtensa' here:
https://github.com/zephyrproject-rtos/sdk-ng/releases/tag/v0.17.0

What process should our customer follow in order to add their product to the toolchain list ?

**Comment by stephanosio** [comment, marked as answer]
They need to submit a PR adding the toolchain. See https://github.com/zephyrproject-rtos/sdk-ng/pull/842 for example.
